module4/model/src/main/java/com/epam/esm/constant/Regex.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.constant;
+
+public final class Regex {
+    public static final String USER_NAME = "[A-Za-zА-Яа-яЁё \\-]{1,45}";
+    public static final String USER_SURNAME = "[A-Za-zА-Яа-яЁё \\-]{1,45}";
+    public static final String PASSWORD = "[A-Za-zА-Яа-яЁё0-9!?@#$%^&*()\\-_+:;,.]{6,256}";
```

Comment:
https://github.com/ViktoriaShamuradova/MJS-School/pull/4#discussion_r608537673 and pay attention to the pattern

Reply:
done